addons/test_mail/tests/test_message_post.py

```diff
@@ -1001,6 +1001,68 @@ def test_message_post_w_attachments_on_main_attachment_model(self):
                         ]
         )
 
+    @mute_logger('odoo.addons.mail.models.mail_mail', 'odoo.addons.mail.models.mail_message_schedule')
+    def test_message_post_w_attachments_filtering(self):
```

Comment:
:socks: should be defined before 'test_message_post_w_attachments_on_main_attachment_model' (alphabetical ordering = execution order = cool :) )